### PR TITLE
Adding account settings section for profile page

### PR DESCRIPTION
**Pull request by maher-suleyman**
PTAL

### CODE REVIEW

**Review by basil-kawak — APPROVED** [review]
LGTM

File: src/containers/profile/ProfileSections/AccountSettings/AccountSettings.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import UpdatePassword from "./AccountSettingsSections/UpdatePassword";
+import DeleteAccount from "./AccountSettingsSections/DeleteAccount";
+
+const AccountSettings = (props) => {
+  const { id, auth } = props;
+
+  return (
+    <div className="accountSettings">
+      <ProfileInfo id={id} auth={auth} />
+      {auth.uid === id && (
```
Comment (louisrli) [inline src/containers/profile/ProfileSections/AccountSettings/AccountSettings.jsx:16]: why does the id in the props need to be checked against auth.uid? is someone else looking at someone else's auth settings?

Comment (maher-suleyman) [reply src/containers/profile/ProfileSections/AccountSettings/AccountSettings.jsx:16]: yes, this condition will prevent the admins from accessing a user's personal page (either the user was a subscriber, author, or another admin)

**Comment by maher-suleyman** [comment]
PTAL

File: src/containers/profile/ProfileSections/AccountSettings/AccountSettings.jsx (hunk 1 above)
Comment (louisrli) [reply src/containers/profile/ProfileSections/AccountSettings/AccountSettings.jsx:16]: ok. just so you know, this is fine for now but in a real app we can't do it like this, the check needs to be serverside. (you should look at firebase security rules -- they allow you to do this).

the reason is that we have access to javascript code in the client side, so i could load the site, delete this if-condition, and access this anyway

**Review by louisrli — APPROVED** [review]
lgtm

**Pull request merged**